tests/test_updater_top_level_update.py

```diff
@@ -306,6 +308,69 @@ def test_new_timestamp_unsigned(self) -> None:
 
         self._assert_files_exist([Root.type])
 
+    @patch.object(datetime, "datetime", wraps=datetime.datetime)
+    def test_expired_timestamp_version_rollback(self, mock_time: Mock) -> None:
+
+        now = datetime.datetime.utcnow()
+        self.sim.timestamp.expires = now.replace(
+            microsecond=0
+        ) + datetime.timedelta(days=7)
```

Comment:
I think I suggested this (since it's what repositorysimulator does) but in fact the `expires` setter nowadays handles the microseconds, so I think this works for all cases where the setter is used:

```suggestion
        self.sim.timestamp.expires = now + datetime.timedelta(days=7)
```